model/service/TestCenterFormService.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ *
+ *
+ */
+namespace oat\taoOffline\model\service;
+
+use common_exception_Error;
+use core_kernel_classes_Class;
+use core_kernel_classes_Resource;
+use oat\oatbox\service\ConfigurableService;
+use oat\taoOffline\model\form\TestCenterForm;
+use oat\taoSync\model\synchronizer\custom\byOrganisationId\testcenter\TestCenterByOrganisationId;
+use tao_helpers_form_FormContainer as FormContainer;
+
+class TestCenterFormService extends ConfigurableService
```

Comment:
Why can't you extend default service?
I mean :

```
class TaoOfflineTestCenterFormService extends oat\taoTestCenter\model\TestCenterFormService
```

then the configuration file will have clean declaration : 
**TestCenterFormService.conf.php**
```
return TaoOfflineTestCenterFormService
```

Reply:
renamed to TaoOfflineTestCenterFormService